server/views/templates/article.njk

```diff
@@ -1,33 +1,60 @@
 {% extends "pages/article.njk" %}
 
-{% block articleBody %}
-  <div class="body-content">
-    {% for bodyPart in bodyParts %}
-      <div class="grid grid--dividers grid--flush-dividers">
-      {% if bodyPart.copy %}
-        <div class="grid__cell grid__cell--xl6 grid__cell--shift-xl1 grid__cell--l7 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4">
-          {{ bodyPart.copy | safe }}
-        </div>
-        {% if bodyPart.figure %}
-          <div class="grid__cell grid__cell--xl4 grid__cell--l5 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4">
-            {% component 'figure', {
-              type: 'picture',
-              sources: bodyPart.figure.sources,
-              caption: bodyPart.figure.caption,
-              modifiers: ['sticky'],
-              icon: bodyPart.figure.icon } %}
+{% block body %}
+<div class="row row--lead row--lead--l">
+  <div class="container">
+    <div class="grid">
+      <div class="grid__cell grid__cell--shift-l1">
+        {% component 'page-description', {title: headline, modifiers: ['a']} %}
+      </div>
+    </div>
+  </div>
+</div>
+{% for media in mainMedia %}
+  {% if media.weighting == 'leading' %}
+    {% component "media-item", { media: media } %}
+  {% endif %}
+{% endfor %}
+
+<div class="row row--cream">
+  <div class="container">
+    <div class="grid">
+      <div class="grid__cell grid__cell--xl6 grid__cell--shift-xl1 grid__cell--l7 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4">
+        {% component 'author', {
+          name: author.name,
+            twitterHandle: author.twitterHandle,
+          image: author.image
+        } %}
+      </div>
+    </div>
+    {% block articleBody %}
+      <div class="body-content">
+        <div class="grid grid--dividers grid--flush-dividers">
+          <div class="grid__cell grid__cell--xl6 grid__cell--shift-xl1 grid__cell--l7 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4 article__main">             
+            {% for bodyPart in bodyParts %}
+                {% if bodyPart.copy %}
+                    {{ bodyPart.copy | safe }}
+                    {% if bodyPart.figure %}
+                        {% component 'figure', {
+                          type: 'picture',
+                          sources: bodyPart.figure.sources,
+                          caption: bodyPart.figure.caption,
+                          modifiers: ['sticky'],
+                          icon: bodyPart.figure.icon } %}
+                    {% endif %}
+                  {% elif bodyPart.figure %}
+                      {% component 'figure', {
+                        type: 'picture',
+                        sources: bodyPart.figure.sources,
+                        caption: bodyPart.figure.caption,
+                        modifiers: ['full-width'],
+                        icon: bodyPart.figure.icon } %}
+                  {% endif %}
+            {% endfor %}
           </div>
```

Comment:
It'd be great to get this into a component (I can't think of a name, as sticky is only applicable to desktop, that feels wrong), and then we can feed it directly into the article page, we just won't have any live content to feed it, but could feed it with the `article.config.yml` for cardigan.

Sorry for the repeat, just writing it down for posterity.